ui/components/AppLayout.tsx

```diff
@@ -28,7 +37,14 @@ export function AppLayout({
     >
       {/*<HelpContainer>*/}
       <ClusterDrawerProvider>
-        <ClusterDrawer>{children}</ClusterDrawer>
+        {isValidKafkaId ? (
+          <ReconciliationProvider kafkaId={kafkaId}>
+            <ReconciliationPausedBanner kafkaId={kafkaId} />
+            <ClusterDrawer>{children}</ClusterDrawer>
+          </ReconciliationProvider>
+        ) : (
+          <></>
+        )}
```

Comment:
This change allows the list of multiple Kafkas to still appear, but blocks the banner if there is no `kafkaId`.

```suggestion
        <ReconciliationProvider kafkaId={kafkaId ?? ""}>
          {isValidKafkaId && (
            <ReconciliationPausedBanner kafkaId={kafkaId} />
          )}
          <ClusterDrawer>{children}</ClusterDrawer>
        </ReconciliationProvider>
```